spring-cloud-aws-core/src/main/java/org/springframework/cloud/aws/core/SpringCloudClientConfiguration.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.aws.core;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.PredefinedClientConfigurations;
+
+/**
+ * @author Eddú Meléndez
+ */
+public final class SpringCloudClientConfiguration {
+
+	private static final String NAME = "spring-cloud-aws";
+
+	private static final String VERSION = "2.3.3";
```

Comment:
I think you meant `2.2.3` here? Would be nice if we could populate this variable from Maven version.

Reply:
good catch! Agree, we can have that value from maven version. I will try to solve it and send another PR

Reply:
In 2.2.3 I think it can stay as it is, perhaps another PR makes sense in 2.3 as there are quite a few changes already.